pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -980,6 +992,17 @@ impl ImageLayerWriter {
         self.inner.as_mut().unwrap().put_image(key, img, ctx).await
     }
 
+    #[cfg(test)]
+    /// Estimated size of the image layer. Currently, this is the blob (compressed) size without the index.
+    pub(crate) fn estimated_size(&self) -> u64 {
```

Comment:
in my unmerged PR, I've added a slightly more sophisticated function that adds the size of the index as well as the summary size: https://github.com/neondatabase/neon/pull/7879/files#diff-bb89836c560ca839968572f69cfbfaf2c7f72429e9c909e9c6329010b96ece53R1102 . maybe use that one?

Reply:
sounds good, I'll change the estimation function

Reply:
https://github.com/neondatabase/neon/pull/8574/commits/05b6c558b7f526897c44a7b744b8f2d9d2bacf73